c++/src/ColumnWriter.cc

```diff
@@ -750,6 +781,12 @@ namespace orc {
     rleEncoder_->recordPosition(rowIndexPosition.get());
   }
 
+  template <typename BatchType>
+  void BooleanColumnWriter<BatchType>::finishStreams() {
+    ColumnWriter::finishStreams();
+    rleEncoder_->finishEncode();
```

Comment:
Add a comment for boolean type. Some trailing boolean bits will be in the next compression block.